fhir-persistence/src/main/java/com/ibm/fhir/persistence/FHIRPersistenceTransaction.java

```diff
@@ -12,33 +12,36 @@
  * This interface represents a transaction within the FHIR persistence layer.
  */
 public interface FHIRPersistenceTransaction {
-    
+
     /**
      * Returns true iff an active transaction exists within the current thread's context.
      */
     boolean isActive() throws FHIRPersistenceException;
-    
+
     /**
      * Begin a new transaction on the current thread.
      * @throws Exception
      */
     void begin() throws FHIRPersistenceException;
-    
+
     /**
      * Commit the current thread's transaction.
      * @throws Exception
      */
     void commit() throws FHIRPersistenceException;
-    
+
+
+    void closeConnection() throws FHIRPersistenceException;
```

Comment:
I don't think close Connection should be in the FHIRPersistenceTransaction

Reply:
then I need a new tag/field in the implementation class, this should also work.